src/textual/color.py

```diff
@@ -329,6 +339,19 @@ def parse(cls, color_text: str | Color) -> Color:
                 clamp(int(float_b), 0, 255),
                 clamp(float_a, 0.0, 1.0),
             )
+        elif hsl is not None:
+            h, s, l = hsl.split(",")
+            h = clamp(float(h), 0, 360) / 360
```

Comment:
I'm not sure that the clamping is the right thing to do here. An `h` of -10 shouldn't be 0, it should be 350.

```suggestion
            h = float(h) % 360
```

Suggest using different names for the strs and the floats. I suspect Mypy might complain about this.

Reply:
Good point!

Reply:
Still needed to `/ 360` at the end